### PR TITLE
Make default upload parallel params 1

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
alleviates problem of #540 as well as joblib seg faults when running the GUIDE with default values, but doesn't actually solve underlying problem (which is unclear what the cause even is at this point...)

### CODE REVIEW

**Comment by garrettmflynn** [comment]
For this to truly take effect, we'd still want to (1) change the default on the Python end, as well as (2) stop populating the actual value of the form to the maximum value when the CPU information is returned from the server.

I'll handle this quick

**Comment by garrettmflynn** [comment]
Should be good to go. Also added a caution message so users understand the consequences of bumping this value.

File: schemas/dandi-upload.schema.ts [hunk 1]
```diff
@@ -6,12 +6,17 @@ import { isStaging } from '../src/renderer/src/stories/pages/uploads/utils'
...
 
+
 const schema = structuredClone(upload)
 const idSchema = schema.properties.dandiset as any
 Object.assign(idSchema, {
     strict: false
 })
 
+const additionalSettings = schema.properties.additional_settings.properties as any
+additionalSettings.number_of_jobs.description = additionalSettings.number_of_threads.description = '<b>⚠️</b> Setting this value has resulted in unpreditable behaviors (runaway processes, segmentation errors, etc.) on several systems</small>'
```
Comment (CodyCBakerPhD) [inline schemas/dandi-upload.schema.ts:17]:
```suggestion
additionalSettings.number_of_jobs.description = additionalSettings.number_of_threads.description = '<b>⚠️</b> Setting this value to more than 1 has resulted in unpredictable behaviors (runaway processes, segmentation faults, etc.) on several systems</small>'
```

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn Small adjustment to displayed warning text, otherwise LGTM (but you need to approve since it's technically my PR lol)

**Review by garrettmflynn — APPROVED** [review]
Awesome. Thanks for updating.

**Pull request merged**